command/agent/check.go

```diff
@@ -6,12 +6,14 @@ import (
 	"log"
 	"net"
 	"net/http"
+	"os"
 	"os/exec"
 	"sync"
 	"syscall"
 	"time"
 
 	"github.com/armon/circbuf"
+	docker "github.com/fsouza/go-dockerclient"
```

Comment:
We're using this in Nomad and Terraform. The code and docs are great and it's well-supported. :+1: